### ISSUE TITLE
AccessibilityInfo::announceForAccessibility fails to announce

### ISSUE THREAD

**Issue by acoates-ms** [microsoft/react-native-windows#5908]
The code to attempt to get an automation peer for the current window calls `FrameworkElementAutomationPeer::FromElement` on an element, but it always returns null.

We should work out how to successfully get a peer from XAML.

(This code will be checked in with #5849)

**Comment by chiaramooney** [comment]
Would advocate to move this issue to 0.65 target. Spoke with Keith, Jon, and heard from accessibility team and each were unclear whether Windows can support the functionality of this API. Current proposed solution is a bit of a hack and would force system into behavior outside of its designed workflow.
